_translator-files/po/nl/Include-Server-Commands.po

```diff
@@ -6,47 +6,43 @@
-"X-Generator: Poedit 3.0\n"
+"X-Generator: Poedit 3.1.1\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:2
-#, fuzzy, no-wrap
-#| msgid "`-d`  or `--discononquit`   Disconnect all clients on quit. Normally, when a server is stopped or restarted, any clients that have not used their \"Disconnect\" buttons will re-establish connection when the server comes back up again. Using this option forces clients to manually re-establish their connections to the server.  \n"
+#, no-wrap
 msgid "`-d`  or `--discononquit`   Disconnect all Clients on quit. Normally, when a Server is stopped or restarted, any Clients that have not used their \"Disconnect\" buttons will re-establish connection when the Server comes back up again. Using this option forces Clients to manually re-establish their connections to the Server.  \n"
 msgstr "`-d`  of `--discononquit`   Verbreek de verbinding met alle clients bij afsluiten. Normaal gesproken, wanneer een server wordt gestopt of opnieuw wordt opgestart, zullen alle clients die hun \"Verbinding verbreken\"-knop niet hebben gebruikt, de verbinding herstellen wanneer de server weer actief is. Als je deze optie gebruikt, worden clients gedwongen hun verbinding met de server handmatig opnieuw tot stand te brengen.  \n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:3
-#, fuzzy, no-wrap
-#| msgid "`-e`  or `--directoryserver`  Register the server on a directory (e.g. to set its genre (see also `-o`)). See [server types](#server-types) for further information.\n"
+#, no-wrap
 msgid "`-e`  or `--directoryserver`  Register the Server on a Directory (e.g. to set its genre (see also `-o`)). See [Server Types](#server-types) for further information.\n"
 msgstr "`-e`  of `--directoryserver`  Registreer de server op een adresboek (bijv. om het genre in te stellen (zie ook `-o`)). Zie [servertypen](#servertypen) voor meer informatie.\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:4
-#, fuzzy, no-wrap
-#| msgid "`--directoryfile` Enable directory to remember registered servers even if the directory is restarted. You'll need to specify a path to a writable file location in which server information is stored.\n"
+#, no-wrap
 msgid "`--directoryfile` Remember registered Servers even if the Directory is restarted. Directory Servers only. See [this guide](Custom-Directories) for further information. \n"
-msgstr "`--directoryfile` Activeer het adresboek om geregistreerde servers te onthouden, zelfs als het adresboek opnieuw wordt gestart. Je moet een pad opgeven naar een beschrijfbare bestandslocatie waarin de serverinformatie wordt opgeslagen.\n"
+msgstr "`--directoryfile` Activeer het adresboek om geregistreerde servers te onthouden, zelfs als het adresboek opnieuw wordt gestart. Je moet een pad opgeven naar een beschrijfbare bestandslocatie waarin de serverinformatie wordt opgeslagen. Zie [deze handleiding](Custom-Directories) voor meer informatie. \n"
```

Comment:
The Dutch has
> `Je moet een pad opgeven naar een beschrijfbare bestandslocatie waarin de serverinformatie wordt opgeslagen.`
but the English does _not_ say
> `You must provide a path to a writable file location where the server information will be stored.`

Technically correct but diversions from the original text become hard to maintain.  Any such additions should be made as changes to the English _first_ and then included in all translations.

Reply:
Looks like I missed a part of the English changes here. Thanks for spotting this!